source/nvwave.py

```diff
@@ -119,6 +119,9 @@ def _winmm_errcheck(res, func, args):
 	func.errcheck = _winmm_errcheck
 
 
+def _isDebug():

```

Comment:
Many of the other similar methods are called `_isDebug` so I understand why you followed that pattern, but it makes it hard to find all the places where a certain category of logging is made. I'd prefer a more descriptive name:
```suggestion
def _isDebugForNvWave():
```

Reply:
> I'd prefer a more descriptive name

I agree with you. I have committed your suggestion and made a second commit with other calls renamed the same way. I have done this in 2 commits because I do not know how to do modifications in a single commit.